.changeset/pretty-otters-whisper.md

```diff
@@ -0,0 +1,69 @@
+---
+'@backstage/plugin-techdocs-node': minor
```

Comment:
You'll be adding a config.d.ts change (see other comment below), which means you'll want to add a second line here for `@backstage/plugin-techdocs-backen: minor` too